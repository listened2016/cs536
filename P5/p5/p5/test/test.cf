struct Point {
    int x;
    int y;
};
int f(int x, bool b) {
	return 0;
}
void g() {
    int a;
    bool b;
    struct Point p;
    p.x = a;
    b = a == 3;
    f(a + p.y*2, b);
    g();
}

int x;
int b;

struct Node {
   struct Point pt;
};
bool z;
int createNode(int a) {
    struct Node n;
    int x;
    int y;
    int pt;
    n.pt.x = 1;
    x=(n.pt.y+1);
    n.pt.x = a;
    n.pt.x = b;
    if (b == -1) { 
        x = 4+3*5-y; 
        while (true) { 
            y = y*2+x; 
        } 
    } 
    else {
        x = 0;
    }
}

bool m(int x, bool b) {
    if (z == false) {
    	int l;
    }
    else {
    }
    return true;
}

struct struct1{
       int foo;
};

struct struct2{
        struct struct1 s;
};
struct struct3{
       struct struct2 v;
};

int test()
{
 struct struct3 z;
 z.v.s.foo = 100;
}
