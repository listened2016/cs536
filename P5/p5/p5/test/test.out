struct Point{
    int x;
    int y;
};

int f(int x, bool b) {
    return 0;
}

void g() {
    int a;
    bool b;
    struct Point p;
    p(Point).x(int) = a(int);
    b(bool) = (a(int) == 3);
    f(int,bool->int)((a(int) + (p(Point).y(int) * 2)), b(bool));
    g(->void)();
}

int x;
int b;
struct Node{
    struct Point pt;
};

bool z;
int createNode(int a) {
    struct Node n;
    int x;
    int y;
    int pt;
    n(Node).pt(Point).x(int) = 1;
    x(int) = (n(Node).pt(Point).y(int) + 1);
    n(Node).pt(Point).x(int) = a(int);
    n(Node).pt(Point).x(int) = b(int);
    if ((b(int) == (-1))) {
        x(int) = ((4 + (3 * 5)) - y(int));
        while (true) {
            y(int) = ((y(int) * 2) + x(int));
        }
    }
    else {
        x(int) = 0;
    }
}

bool m(int x, bool b) {
    if ((z(bool) == false)) {
        int l;
    }
    else {
    }
    return true;
}

struct struct1{
    int foo;
};

struct struct2{
    struct struct1 s;
};

struct struct3{
    struct struct2 v;
};

int test() {
    struct struct3 z;
    z(struct3).v(struct2).s(struct1).foo(int) = 100;
}

struct Point{
    int x;
    int y;
};

int f(int x, bool b) {
    return 0;
}

void g() {
    int a;
    bool b;
    struct Point p;
    p(Point).x(int) = a(int);
    b(bool) = (a(int) == 3);
    f(int,bool->int)((a(int) + (p(Point).y(int) * 2)), b(bool));
    g(->void)();
}

int x;
int b;
struct Node{
    struct Point pt;
};

bool z;
int createNode(int a) {
    struct Node n;
    int x;
    int y;
    int pt;
    n(Node).pt(Point).x(int) = 1;
    x(int) = (n(Node).pt(Point).y(int) + 1);
    n(Node).pt(Point).x(int) = a(int);
    n(Node).pt(Point).x(int) = b(int);
    if ((b(int) == (-1))) {
        x(int) = ((4 + (3 * 5)) - y(int));
        while (true) {
            y(int) = ((y(int) * 2) + x(int));
        }
    }
    else {
        x(int) = 0;
    }
}

bool m(int x, bool b) {
    if ((z(bool) == false)) {
        int l;
    }
    else {
    }
    return true;
}

struct struct1{
    int foo;
};

struct struct2{
    struct struct1 s;
};

struct struct3{
    struct struct2 v;
};

int test() {
    struct struct3 z;
    z(struct3).v(struct2).s(struct1).foo(int) = 100;
}

